src/librustc_trans/back/msvc/mod.rs

```diff
@@ -31,8 +31,18 @@
 //! paths/files is based on Microsoft's logic in their vcvars bat files, but
 //! comments can also be found below leading through the various code paths.
 
+// A simple macro to make this option mess easier to read
+macro_rules! otry {
+    ($expr:expr) => (match $expr {
+        Some(val) => val,
+        None => return None,
+    })
+}
```

Comment:
Clearly this just indicates the need for this sort of macro in libstd. Sure would be nice if the new `?` stuff supported `Option`.